tools/introspection/package.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+    <name>iceoryx_introspection</name>
+    <version>2.0.4</version>
+    <description>Eclipse iceoryx inter-process-communication (IPC) middleware introspection client</description>
+    <maintainer email="iceoryx-oss-support@apex.ai">Eclipse Foundation, Inc.</maintainer>
+    <license>Apache 2.0</license>
+    <url type="website">https://iceoryx.io</url>
+    <url type="bugtracker">https://github.com/eclipse-iceoryx/iceoryx/issues</url>
+    <url type="repository">https://github.com/eclipse-iceoryx/iceoryx</url>
+
+    <buildtool_depend>cmake</buildtool_depend>
+
+    <depend>iceoryx_posh</depend>
+    <depend>ncurses</depend>
```

Comment:
@mossmaurice The rosdep key must be

```suggestion
    <depend>libncurses-dev</depend>
```

It is coming from [rosdistro](https://github.com/ros/rosdistro/blob/master/rosdep/base.yaml#L4255) and can be checked with the command: `rosdep resolve libncurses-dev`.
Sorry for the late change.

Reply:
@dkroenke No worries, but why didn't the ROS CI catch this?

Reply:
@mossmaurice Not an expert on this. The [ROS 2 CI](https://ci.ros2.org/) does not actively check rosdep dependencies it just installs all of them and builds against them.
The real check happens with bloom releases on the [ROS 2 Buildfarm](https://build.ros.org/) where the debian packages are created in a bare metal env and the dependencies from package.xml are evaluated. But that happens after merging and tagging the release...

Reply:
@dkroenke Got it, fixed.